pset-functions.js

```diff
@@ -91,6 +117,22 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const phrase1 = "olly olly in come free"
+const phrase2 = "Baby shark, doo doo doo doo doo doo"
+const phrase3 = "Humpty Dumpty sat on a wall Humpty Dumpty had a great fall"
+
+
+
+function wordCount(str) {
+    return str.replace(/,/g, "").split(" ").reduce(function(count, word) {
+      count[word] = count.hasOwnProperty(word) ? count[word] + 1 : 1
+      return count
+    }, {})
+}
+
+console.log(wordCount(phrase1))
+console.log(wordCount(phrase2))
+console.log(wordCount(phrase3))
```

Comment:
@rlou88 Nice job coming up with this solution. Good job chaining the methods on line 127 and using the conditional ternary operator to either create the word and set the value to 1 if it doesn't exist, and increment the value of the word by 1.

Take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L103)